### DISCUSSION TITLE
Problem with yolov5

### DISCUSSION THREAD

**Discussion by QWERTYpy** [pyinstaller/pyinstaller#7279] [Help]
Good afternoon.
Tell me please.
There is a script:
_import yolov5
from init_const import dict_conf
model_path = dict_conf['model_path']
device = "cpu"
yolov5 = yolov5.YOLOv5(model_path, device)_

launching:
_pyinstaller --onefile test.py_

everything is fine:
_Building EXE from EXE-00.toc completed successfully._

But when running exe:
FileNotFoundError: [WinError 3] Системе не удается найти указанный путь: 'C:\\Users\\...\\AppData\\Local\\Temp\\_MEI93042\\yolov5\\utils\\general.pyc'

In the temporary folder yolov5 does not even appear.

**Comment by rokm** [comment]
See https://github.com/pyinstaller/pyinstaller/issues/7268#issuecomment-1324104624.

**Reply by QWERTYpy** [reply]
thank you so much.
